cmake/dependencies/openPMD.cmake

```diff
@@ -11,7 +11,7 @@ function(find_openpmd)
         set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
 
         # see https://openpmd-api.readthedocs.io/en/0.14.2/dev/buildoptions.html
-        set(openPMD_USE_MPI         ${HiPACE_MPI} CACHE INTERNAL "")
+        set(openPMD_USE_MPI         OFF           CACHE INTERNAL "")
```

Comment:
Let's make this an option.

The reason for that is: let's assume you want to use HiPACE++ in a software environment with other BLAST codes that use openPMD, too. Some might depend on a parallel HDF5. But HDF5 unfortunately cannot be installed in two binary variants at the same time in environments like conda.

Similarly, an HPC system might only provide us with a parallel HDF5 (or only a serial HDF5) module.

In such situations, we want to control the parallelism of the I/O layer, even if we only use it serially.

Simple solution that I prefer:
we add an `option(...)`.

Fancy solution:
we do a `find_package(HDF5 ...)` and decide based on this (but this is more involved and does not apply to other backends like ADIOS2)

Reply:
I agree, let's do the simple solution. Will update the PR when I have time :)
Thanks!